driver-core/src/main/com/mongodb/internal/connection/ExtendedAsynchronousByteChannel.java

```diff
@@ -17,7 +17,7 @@
  * https://github.com/marianobarrios/tls-channel
  */
 
-package com.mongodb.internal.connection.tlschannel.async;
+package com.mongodb.internal.connection;
```

Comment:
Funny, I thought about moving this up when AsynchronousSocketChannelStream started to use it, but then didn't do it.